code/__defines/mobs.dm

```diff
@@ -330,7 +330,7 @@
 #define SPECIES_SHADOW			"Shadow"
 #define SPECIES_SKELETON		"Skeleton"
 #define SPECIES_GOLEM			"Golem"
-#define SPECIES_EVENT1			"X Occursus"
+#define SPECIES_EVENT1			"X Anomalous"
```

Comment:
At first I removed EVENT1 entirely, but then I realized it's used in quite a few places in the code, as was EVENT2 and EVENT3.

It required less touching the code to just swap EVENT1 over to the same as EVENT2 as opposed to removing either EVENT1/2/3 entirely and only having two EVENT species.

Reply:
Okay, but why not add the name back in properly? Because now in code there are two species with same name and game really won't like it. I mean, I know our staff pretty much doesn't use event species ever; since they require var-based customizations to work, but it is better to avoid that scenario in the first place.

Reply:
Fair point, let me go ahead and change their name back. Thinking about it, it shouldn't really affect much if I leave the name as it was.